pyqtgraph/graphicsItems/TargetItem.py

```diff
@@ -1,132 +1,429 @@
+from math import atan2, pi
+
 from ..Qt import QtGui, QtCore
-import numpy as np
 from ..Point import Point
 from .. import functions as fn
 from .GraphicsObject import GraphicsObject
+from .UIGraphicsItem import UIGraphicsItem
 from .TextItem import TextItem
+from .ScatterPlotItem import Symbols
+from .ViewBox import ViewBox
+import string
 
 
-class TargetItem(GraphicsObject):
+class TargetItem(UIGraphicsItem):
     """Draws a draggable target symbol (circle plus crosshair).
 
     The size of TargetItem will remain fixed on screen even as the view is zoomed.
     Includes an optional text label.
     """
-    sigDragged = QtCore.Signal(object)
 
-    def __init__(self, movable=True, radii=(5, 10, 10), pen=(255, 255, 0), brush=(0, 0, 255, 100)):
-        GraphicsObject.__init__(self)
-        self._bounds = None
-        self._radii = radii
-        self._picture = None
+    sigPositionChanged = QtCore.Signal(object)
+    sigPositionChangeFinished = QtCore.Signal(object)
```

Comment:
These are well named signals! Do we need to do a deprecation dance with `sigDragged`?

Reply:
I admit, I had to lookup `sigDragged`, didn't know of that signal off the top of my head.  I'm not opposed to that; it arguably makes a bit more sense with `ROI.py` ...but for InfiniteLine, I would think `sigPositionChanged` would be sufficient (and frankly, I think `positionChanged` would be even better as `positionChanged` is the name of a signal is some upstream objects that are more or less equivalent.

Reply:
Oh, sorry, I was confusing.

I mean that the previous code used `sigDragged` instead of `sigPositionChanged`, and we should keep the previous interface around for at least one version, but issue a deprecation warning. Same could be applied to `setLabelAngle`.

Reply:
Ahhh right I keep forgetting that this PR replaces something, even though it was completely undocumented.

I'll go back in and insert the original methods and throw some deprecation warnings.

Reply:
Fixed with 5ae0175cbf0ee0622d58cd72826b6358077c30c6